src/bvals/comms/boundary_communication.cpp

```diff
@@ -171,15 +178,17 @@ SendBoundBufs<BoundaryType::flxcor_send>(std::shared_ptr<MeshData<Real>> &);
 
 template <BoundaryType bound_type>
 TaskStatus StartReceiveBoundBufs(std::shared_ptr<MeshData<Real>> &md) {
+  return TaskStatus::complete;
```

Comment:
Should this be here, or is it a debugging leftover?

Reply:
done